sdk/messaging/azservicebus/internal/tracing/attributes_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tracing
+
+// write unit tests for go
+import (
+	"testing"
+	"time"
+
+	"github.com/Azure/go-amqp"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetMessageSpanAttributes(t *testing.T) {
+	messageId := "message-id"
+	correlationId := "correlation-id"
+
+	testCases := []struct {
+		name     string
+		message  *amqp.Message
+		expected []Attribute
+	}{
+		{
+			name:     "empty message",
+			message:  &amqp.Message{},
+			expected: []Attribute{},
```

Comment:
Mind adding one where 'expected' is 'nil' as well? I expect the same result as this one, of course.